homeassistant/components/fjaraskupan/binary_sensor.py

```diff
@@ -82,7 +83,7 @@ def __init__(
 
         self._attr_unique_id = f"{device.address}-{entity_description.key}"
         self._attr_device_info = device_info
-        self._attr_name = f"{device_info['name']} {entity_description.name}"
+        self._attr_name = entity_description.name
```

Comment:
This is the default behavior, this line can be removed

Reply:
```suggestion
```